tests/unittests/tests-checksum/tests-checksum-fletcher16.c

```diff
@@ -14,6 +14,10 @@
 
 #include "tests-checksum.h"
 
+/* String is longer than 20 so it checks the wrap-around property of
+ * fletcher16 */
+static const unsigned char wrap_around_data[] = "u9k89rm88z58f3hGxTfgG3BnyKnyRP";
```

Comment:
Well, this won't work with properly on non-ASCII platforms, but I guess RIOT won't be ported to 30 year old IBM mainframes anytime soon, so this non-portability probably OK. :)